selenoid.go

```diff
@@ -281,6 +281,22 @@ func create(w http.ResponseWriter, r *http.Request) {
 			w.WriteHeader(resp.StatusCode)
 		}
 	} else {
+		body, err := io.ReadAll(resp.Body)
+		if err != nil {
+			log.Printf("[%d] [ERROR_READING_RESPONSE] [%v]", requestId, err)
+			queue.Drop()
+			cancel()
+			return
+		}
+		newBody, err := addSeCdpCapability(body, r.Host)
+		if err != nil {
+			log.Printf("[%d] [ERROR_ADDING_SE_CDP_CAPABILIITY] [%v]", requestId, err)
+			queue.Drop()
+			cancel()
+			return
+		}
+		resp.Body = io.NopCloser(bytes.NewReader(newBody))
+		resp.ContentLength = int64(len(newBody))
 		tee := io.TeeReader(resp.Body, w)
```

Comment:
Probably need to use `body` or `newBody` instead of `resp.Body` here to prevent using already read io.Reader.

Reply:
Or you could simply get ID field from already parsed body.